tests/integration/conftest.py

```
@@ -454,7 +475,10 @@ def configure_environment(get_configuration, request):
 
     # Remove created folders (parents)
     if sys.platform == 'win32':
-        control_service('stop')
+        try:
+            control_service('stop')
+        except ValueError:
+            pass
```

Don't silence the error.